eosdatadistribution/producers/usb.py

```diff
@@ -56,7 +56,7 @@ def mount_removed_cb(monitor, mount, store):
     pprint.pprint([store.remove_name(n) for p, n in store.producers.items() if p.startswith(p)])
 
 
-if __name__ == '__main__':
+def main():
```

Comment:
I’ve re-read on why the cargo-culting was correct, and having a `main()` method is the [recommended approach](https://packaging.python.org/distributing/#scripts) because it is [more testable](http://python-packaging.readthedocs.io/en/latest/command-line-scripts.html#the-console-scripts-entry-point) — the script can be tested by calling the `main()` function from a unit test, without needing to spawn the script itself.